ui/serializers.py

```diff
@@ -7,6 +7,28 @@
 from rest_framework.settings import api_settings
 
 from ui import models, permissions as ui_permissions
+from ui.utils import get_moira_client
+
+
+def validate_moira_lists(lists):
+    """
+    Raise a validation error if any of the moira lists in a list does not exist or is not a mailing list
+
+    Args:
+        lists(list): List of moira lists
+    """
+    bad_lists = []
+    for mlist in lists:
+        moira_client = get_moira_client()
+        if not moira_client.list_exists(mlist.name):
+            bad_lists.append(mlist.name)
+        else:
+            attributes = moira_client.client.service.getListAttributes(mlist.name, moira_client.proxy_id)
+            if not (attributes and 'mailList' in attributes[0]):
```

Comment:
Can you add a test for the case where this expression is false?

Reply:
`test_collection_serializer_validation_nonmail_view_lists`, `test_collection_serializer_validation_nonmail_admin_lists`